config/ember-try.js

```diff
@@ -51,7 +51,17 @@ module.exports = function() {
             'testem': '~2.0.0'
           }
         }
-      }
+      },
+      {
+        name: 'ember-babel-8',
+        npm: {
```

Comment:
Is there a reason why we don't have a `command` in this scenario but we do for the others?

Reply:
Ember test is the default, afaik